source/NVDAObjects/window/winword.py

```diff
@@ -635,6 +635,33 @@ def label(self):
 		return _("spelling: {text}").format(text=text)
 
 
+class WordDocumentReferenceQuickNavItem(WordDocumentCollectionQuickNavItem):
+	def rangeFromCollectionItem(self, item):
+		return item.reference
```

Comment:
please add type hints

Reply:
Done. Note that lazybind Dispatch objects are used here, I don't think there's a way to hint this better ATM.